lib/linguist/heuristics.yml

```diff
@@ -61,6 +61,10 @@ disambiguations:
     - pattern: '\b(?i:(CODEUNIT|PAGE|PAGEEXTENSION|PAGECUSTOMIZATION|DOTNET|ENUM|ENUMEXTENSION|VALUE|QUERY|REPORT|TABLE|TABLEEXTENSION|XMLPORT|PROFILE|CONTROLADDIN|REPORTEXTENSION|INTERFACE|PERMISSIONSET|PERMISSIONSETEXTENSION|ENTITLEMENT))\b'
   # Open-ended fallback to Perl AutoLoader
   - language: Perl
+- extensions: ['.app']
+  rules:
+  - language: Erlang
+    pattern: '^{\s*(?:application|''application'')\s*,\s*(?:[a-z]+[a-z\d_@]*|''[\s\S]+'')\s*,\s*\[[\s\S]*\]\s*}\.$'
```

Comment:
This doesn't look right to me with all those unescaped quots and `\s` and feels like it could be improved too.

@Alhadis over to you 🙇

Reply:
An application specification is an Erlang term, more specifically, a triple. White space is not significant in Erlang and any of the terms can be padded with an arbitrary number of white space characters.

Reply:
@Alhadis when you've got a mo, could you take a squizz as this regex. Thanks 🙇

Reply:
@lildude Done. 👍